htdocs/src/AppBundle/Controller/ChangelogController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use League\CommonMark\CommonMarkConverter;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+class ChangelogController extends Controller
+{
+    /**
+     * @Route("/changelog")
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function indexAction()
+    {   $converter = new CommonMarkConverter();
```

Comment:
you should define a service for the converter and use this instead

Reply:
done :-)